src/app/promoScreen/promoScreen.module.css

```diff
@@ -0,0 +1,25 @@
+.gradientBall {
+    position: 'absolute';
```

Comment:
this is css now not react inline styles so need to change these:
e.g.
from height: '1500px'; to height: 1500px;